pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computebackendservice/globalcomputebackendserviceiap/dependencies.yaml

```diff
@@ -62,9 +62,12 @@ metadata:
 apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
 kind: Project
 metadata:
+  annotations:
+    cnrm.cloud.google.com/deletion-policy: "abandon"
   name: iapclient-${uniqueId}
 spec:
-  name: ComputeProjectMetadata
+  resourceID: ${TEST_DEPENDENT_ORG_PROJECT_ID}
```

Comment:
As before, a comment on why we need a "fixed" project / what's special about this project would be great!

Reply:
@justinsb how detailed the information are you looking for? The change is purely based on internal reasons and I feel `using a fixed project due to internal reasons" is the only thing we can comment here to avoid exposing internal infrastructure.